src/User/Bootstrap.php

```diff
@@ -154,6 +154,12 @@ function () use ($model) {
                 $di->set(Search\RoleSearch::class);
             }
 
+            if (Yii::$app->getModule('user')->searchUsersInLdap) {
+                if (!class_exists('kartik\typeahead\Typeahead')) {
+                    throw new InvalidConfigException('The kartik-v/yii2-widget-typeahead library must be defined when searchUsersInLdap is true.');
```

Comment:
_must be **installed**_ rather than _defined_

Reply:
also here https://github.com/2amigos/yii2-usuario/pull/543/files#diff-3af9b9b60ce0ece19bdfd796fed5974821179bdc8da2278e7f5d0c8b4b6671eeR21 is a new dependency not installed, nor documented